docs/features/techdocs/architecture.md

```diff
@@ -58,12 +58,12 @@ Similar to how it is done in the Basic setup, the TechDocs Reader requests
 your configured storage solution for the necessary files and returns them to
 TechDocs Reader.
 
-Note about caching: We have noticed internally that some storage providers can
-be quite slow, which is why we are recommending a cache that sits between the
-TechDocs Reader and the Storage.
-
-_Feel free to suggest better ideas to us in #docs-like-code channel in Discord
-or via a GitHub issue._
+Depending on your chosen cloud storage provider and its real-world proximity to
+your backend server, there may be a comparably high amount of latency when
+loading TechDocs sites using this deployment approach. If you encounter this,
+you can optionally configure the `techdocs-backend` to cache responses in a
+cache store
+[supported by Backstage](../../overview/architecture-overview.md#cache).
```

Comment:
Have we considered the effect of deployment strategies of the Backstage instance? I.e. if running on multiple nodes (e.g. availability zones), we might want to give a hint on the tradeoff between storage and speed. So in order to maximize speed one might consider caching at the edge, or having one cache per region if storage is a concern

Reply:
Indeed, and this more closely approximates how we run things internally now.  Wanted to keep the blurb here high-ish level since it's about architecture generally and the potential configurations with benefits / tradeoffs get complicated quickly.  Good opportunity to expand on all of this in a "deployment configurations and recommendations" section of its own, though.